dCommon/dEnums/dCommonVars.h

```diff
@@ -27,6 +27,13 @@ constexpr uint32_t highFrameDelta = FRAMES_TO_MS(highFramerate);
 constexpr uint32_t mediumFrameDelta = FRAMES_TO_MS(mediumFramerate);
 constexpr uint32_t lowFrameDelta = FRAMES_TO_MS(lowFramerate);
 
+//========== CLIENT VERSION DEFAULTS ===========
+namespace ClientVersion {
+	constexpr uint16_t major = 1;
+	constexpr uint16_t current = 10;
+	constexpr uint16_t minor = 64;
+}
+
```

Comment:
these do not need to be included in every single source file and can be moved back to their original location.

Reply:
Why not? They're constexpr, so they don't actually exist at runtime. It's effectively the same as something like:

#define MAJOR_VERSION 1
#define CURRENT_VERSION 10
#define MINOR VERSION 64

Reply:
yes but this is a PCH.  Every source file does not need this information in it.

Reply:
Moving it to a new header file